tldw_Server_API/tests/MediaIngestion_NEW/unit/test_audio_transcription.py

```
@@ -50,3 +63,548 @@ def test_convert_to_wav_rejects_invalid_range(tmp_path):
+    header = (
+        "This text was transcribed using whisper model: distil-large-v3\n"
+        "Detected language: en\n\n"
+        "Hello world"
+    )
+    segments = [{"Text": header, "start_seconds": 0.0, "end_seconds": 1.0}]
+    out = strip_whisper_metadata_header(segments)
+    assert out[0]["Text"] == "Hello world"
+
+
+@pytest.mark.unit
+def test_transcribe_audio_uses_safe_default_provider(monkeypatch, tmp_path):
```

Ruff (0.14.5)

Unused function argument: `tmp_path`

(ARG001)